### PR TITLE
added nodeselector in otelCollectorMetrics and otelCollector

### PR DESCRIPTION
**Pull request by medicharlachiranjeevi**

### CODE REVIEW

File: charts/signoz/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: signoz
-version: 0.0.12
```
Comment (prashant-shahi) [inline charts/signoz/Chart.yaml:3]: @medicharlachiranjeevi can you please revert this change? we only bump up the chart version with bunch of changes together. Mostly post new signoz release.

Comment (medicharlachiranjeevi) [reply charts/signoz/Chart.yaml:3]: Sure

**Review by prashant-shahi — APPROVED** [review]
:shipit: LGTM

**Comment by prashant-shahi** [comment]
@medicharlachiranjeevi Thanks for your PR 🎉
